cogs5e/models/initiative/combatant.py

```diff
@@ -251,6 +251,10 @@ def set_group(self, group_name):
                 self.combat.goto_turn(self, True)
             return c_group
 
+    def get_group_name(self):
```

Comment:
Consider making this `get_group` to be more general and using `get_group().name` (with relevant None checks) in the AliasCombatant method